web/src/upload.js

```diff
@@ -437,6 +438,12 @@ export function initialize() {
         mode: "compose",
     });
 
+    $(".app, #navbar-fixed-container").on("dragstart", (event) => {
+        if (event.target.nodeName === "IMG") {
+            drag_drop_img = event.target;
+        }
+    });
```

Comment:
Presumably this should clear `drag_drop_img` in an else clause, right?

Reply:
Yes, makes sense. Updated to reflect the same.